scribble-lib/scribble/private/tag.rkt

```diff
@@ -0,0 +1,39 @@
+#lang scheme/base
+
+;; It might make sense to make these functions public, but since they weren't originally,
+;; I am going to keep them in the private folder for now.
+
+(require racket/list
+         scribble/core
+         "../tag.rkt")
+
+(provide (all-defined-out))
```

Comment:
This program appears to compile:

```
#lang racket/base
(require racket/contract/base)
(provide (contract-out [foo engine?]))
(require racket/engine)
(define foo (engine void))
```

Or are we referring to different problems?